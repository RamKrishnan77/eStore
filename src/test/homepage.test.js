import {shallow} from 'enzyme'
import React from 'react'
import HomePage from '../components/homePage.component'

//Snapshot testing of SignIn using Eznymeâ€™s Full DOM rendering 

it('expect to render homepage component', () => {
    expect(shallow(<HomePage></HomePage>)).toMatchSnapshot()
})