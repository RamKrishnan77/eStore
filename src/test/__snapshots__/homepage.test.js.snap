// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render homepage component 1`] = `
<div
  className="panel1"
>
  <Image
    fluid={true}
    rounded={false}
    roundedCircle={false}
    src="https://i.pinimg.com/originals/fa/45/96/fa4596ad9a9d39901eeb455ed4f74e44.jpg"
    thumbnail={false}
  />
  <Image
    fluid={true}
    rounded={false}
    roundedCircle={false}
    src="https://s3.amazonaws.com/nikeinc/assets/91660/Nike-Air-Zoom-Pulse_2_native_1600.jpg?1573245547"
    thumbnail={false}
  />
  <Image
    fluid={true}
    rounded={false}
    roundedCircle={false}
    src="https://s3.amazonaws.com/nikeinc/assets/93455/nike-air-max-2090-4_rectangle_1600.jpg?1579197133"
    thumbnail={false}
  />
</div>
`;
